src/server.ts

```diff
@@ -27,6 +27,7 @@ import { config } from '@/config-manager';
 const app: express.Express = express();
 
 // START APPLICATION
+
 app.set('view engine', 'ejs');
 app.set('views', __dirname + '/views');
 // Create router

```

Comment:
I had meant a newline here after `app.set('views', __dirname + '/views');` and before `app.set('views', __dirname + '/views');`